seal.go

```diff
@@ -35,6 +37,35 @@ const (
 	Update
 )
 
+type File interface {
```

Comment:
(nitpick) `File` seems to be very generic (plus there is os.File already which may be confusing) and GetBytes() is a bit unexpected for such an interface. Not that I have a better suggestion :/

Reply:
Agree with @mvo5, a less generic name could work better here. And since `GetBytes()` actually gets the entire file contents, perhaps a name more in the `ReadAll()` line could convey this idea better?

Reply:
I've renamed GetBytes() to ReadAll(). I agree about the naming of File, but I couldn't think of a better name for now.